tests/subsys/jwt/src/jwt-test-private.c

```diff
@@ -4,109 +4,130 @@
+	0x7c, 0x4c, 0x49, 0xb7, 0x51, 0x0f, 0x40, 0xf2, 0x7c, 0x46, 0xcc, 0x7f,
+	0x46, 0xb3, 0x78, 0xab, 0x17, 0x02, 0x81, 0x81, 0x00, 0x9d, 0xaa, 0x8d,
+	0x0d, 0xcf, 0x88, 0xb5, 0xe1, 0x5f, 0xfb, 0xc3, 0xf7, 0x8e, 0x91, 0x6e,
+	0xf1, 0x18, 0x77, 0xa2, 0xa6, 0x18, 0xe4, 0x0c, 0x1e, 0xec, 0x8a, 0x7b,
+	0x76, 0x36, 0xed, 0xc1, 0xcf, 0x7b, 0xc8, 0xf2, 0x4d, 0xa9, 0x39, 0x61,
+	0x6d, 0xd4, 0xb0, 0xe2, 0xea, 0x35, 0xaa, 0xdd, 0x11, 0xd8, 0x56, 0x5d,
+	0xae, 0xc6, 0xaf, 0xf8, 0x12, 0x6c, 0x92, 0x31, 0x99, 0x40, 0x93, 0xa4,
+	0xdf, 0xce, 0x96, 0x10, 0xaa, 0x33, 0x18, 0x3b, 0xf5, 0x90, 0x22, 0x68,
+	0xa3, 0x12, 0xe6, 0x13, 0xb3, 0xf6, 0x6c, 0x14, 0x92, 0x04, 0xf6, 0x76,
+	0xe1, 0x13, 0x6b, 0xae, 0x6d, 0x2a, 0x7e, 0x74, 0x52, 0x57, 0x40, 0x16,
+	0x8f, 0x2e, 0x39, 0x8e, 0x57, 0xf9, 0xa9, 0x5c, 0x77, 0xf3, 0xb5, 0xbb,
+	0x08, 0x49, 0xfc, 0xe6, 0x74, 0x5a, 0xf2, 0xcd, 0x5c, 0x38, 0xa1, 0x70,
+	0x29, 0x3b, 0x2f, 0xd5, 0x07, 0x02, 0x81, 0x81, 0x00, 0xa9, 0xe0, 0xa7,
+	0x96, 0x89, 0xe3, 0xde, 0x0d, 0x1c, 0x49, 0x36, 0xe0, 0x77, 0xc2, 0x4e,
+	0xdc, 0xf8, 0x9d, 0x1c, 0x9e, 0xb7, 0xde, 0xa8, 0x3c, 0x1a, 0x03, 0xa2,
+	0x8a, 0xb4, 0xfe, 0x72, 0x3f, 0x3e, 0xe1, 0xc5, 0xc2, 0x06, 0x63, 0xb8,
+	0xfc, 0xe1, 0xad, 0x9a, 0x7c, 0x88, 0x9a, 0x46, 0x2e, 0x75, 0xff, 0x6e,
+	0xe2, 0x2c, 0xd8, 0x92, 0xc5, 0xc0, 0xf6, 0x78, 0x79, 0x46, 0x54, 0x81,
+	0x22, 0xa3, 0xd3, 0x2a, 0xda, 0x71, 0x17, 0x42, 0x41, 0x92, 0xbf, 0xa5,
+	0x48, 0x16, 0x8d, 0xe0, 0xb6, 0xce, 0x23, 0x97, 0x26, 0xe0, 0x91, 0x33,
+	0x8b, 0xc2, 0x91, 0xc9, 0xde, 0xb1, 0x9f, 0xda, 0x4a, 0xc1, 0xb0, 0x9a,
+	0x1e, 0xc8, 0x4a, 0x58, 0x84, 0x4d, 0x1d, 0x2a, 0xf8, 0xed, 0xb0, 0x07,
+	0x9c, 0xc6, 0x0d, 0x31, 0xef, 0x34, 0xa2, 0x35, 0xe9, 0xf0, 0x77, 0x09,
+	0xd0, 0xce, 0xa0, 0x1f, 0xa7, 0x02, 0x81, 0x81, 0x00, 0xb0, 0x9d, 0xd2,
+	0x7f, 0xff, 0xe0, 0x86, 0x3e, 0x5f, 0xfe, 0x83, 0xf6, 0x51, 0xa3, 0x09,
+	0xb2, 0xb6, 0x4a, 0xe0, 0xdf, 0xd9, 0xdd, 0x2a, 0x9f, 0x68, 0x4c, 0xb2,
+	0x4f, 0x12, 0x1d, 0x07, 0xb8, 0x75, 0x70, 0x28, 0x7a, 0x49, 0xaa, 0x56,
+	0x53, 0x7a, 0xc7, 0xbf, 0xbd, 0xea, 0x38, 0x2c, 0xf0, 0x91, 0xf6, 0x71,
+	0x1e, 0x97, 0x10, 0x20, 0x79, 0x2b, 0x24, 0x6c, 0x1c, 0x7b, 0x2e, 0x96,
+	0x8c, 0x72, 0x13, 0xc1, 0x60, 0x7c, 0x41, 0x20, 0x8b, 0x36, 0x4c, 0xe2,
+	0x65, 0x8d, 0xc5, 0x0e, 0x1e, 0x34, 0x01, 0x6d, 0x45, 0x48, 0x86, 0x6e,
+	0x45, 0x43, 0xf4, 0xff, 0xb8, 0xa0, 0xa8, 0x07, 0x49, 0x54, 0x92, 0x0f,
+	0x53, 0xe9, 0x2c, 0xd3, 0xdb, 0x6a, 0x8a, 0xeb, 0xc4, 0x15, 0x51, 0xdf,
+	0x9c, 0x1a, 0x1a, 0x20, 0x8a, 0xed, 0xfe, 0xef, 0xe8, 0xe1, 0x04, 0x14,
+	0x05, 0xfd, 0x71, 0xb0, 0x3e
+};
+
+#else /* CONFIG_JWT_SIGN_ECDSA */
+
+/* Here's how to generate the key in the correct format:
+ * - generate the key using OpenSSL:
+ *   $ openssl ecparam -name prime256v1 -genkey | openssl ec -outform der | xxd -ps
+ * - copy the content to https://lapo.it/asn1js/
+ * - get the first OCTECT STRING, which is the private key, and copy it here
+ */
 
 unsigned char jwt_test_private_der[] = {
```

Comment:
In this case, the clang format output is actively terrible. If you have ideas on improving the config for this one, perhaps that should be proposed. But, not in this PR, just leave it.